cppcache/include/geode/Cache.hpp

```diff
@@ -167,17 +167,17 @@ class APACHE_GEODE_EXPORT Cache : public GeodeCache {
   virtual void readyForEvents();
 
   /**
-   * Creates an authenticated cache using the given user security properties.
-   * Multiple instances with different user properties can be created with a
-   * single client cache.
+   * Creates an authenticated cache using the given user security
+   * properties. Multiple instances with different user properties can be
```

Comment:
Why is this suddenly changing format? Will clang-format choke on this?

Reply:
Not sure how this got changed. Reverting